### PR TITLE
Gh66

### PR DESCRIPTION
**Pull request by Insoleet**
Speed optimizations (https://github.com/harvimt/quamash/issues/66)

### CODE REVIEW

File: quamash/_windows.py [hunk 1]
```diff
@@ -52,9 +52,6 @@ def _after_run_forever(self):
 		self.__event_poller.stop()
 
 
-baseclass = _ProactorEventLoop
```
Comment (harvimt) [inline quamash/_windows.py]: why? this will break stuff?

**Comment by harvimt** [comment]
I made some tweaks in #82 (and I'm gonna try to get the CI tests to pass)

**Comment by harvimt** [comment]
I meant #83

File: quamash/_windows.py (hunk 1 above)
Comment (Insoleet) [reply quamash/_windows.py]: Oh, this leaked from my custom branch ( see this issue https://github.com/harvimt/quamash/issues/45 )

**Pull request merged**